nova/virt/vmwareapi/vmops.py

```diff
@@ -2389,6 +2450,13 @@ def _resize_vm(self, context, instance, vm_ref, flavor, image_meta):
             except Exception:
                 LOG.warning('Could not remove DRS override.',
                             instance=instance)
+
+        # This might come back as None if the flavor doesn require an EVCMode.
```

Comment:
Typo:
```suggestion
        # This might come back as None if the flavor doesn't require an EVCMode.
```

But 81 chars, nooo!

```suggestion
        # This may come back as None if the flavor doesn't require an EVCMode.
```

\o/